gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClient.java

```diff
@@ -25,6 +25,8 @@
       BrokerResponseConsumer<T> responseConsumer,
       Consumer<Throwable> throwableConsumer);
 
+  <T> ActorFuture<BrokerResponse<T>> sendRequest(BrokerRequest<T> request, Duration requestTimeout);
```

Comment:
I reverted the removal of this method after discussing it with @npepinpe.

The method now only exists for a single test, but it was easier just to revert the removal of the method than to rewrite the test. A more thorough refactoring of the BrokerClient is needed anyways.